src/main/org/deidentifier/arx/metric/Metric.java

```diff
@@ -1143,8 +1143,11 @@ protected static MetricDescription getDescription(Metric<?> metric) {
 
     /** Is the metric independent?. */
     private boolean           independent      = false;
+    
+    /** Is the metric monotonic with generalization?. */
+    private boolean           monotonicWithGeneralization    = true;
 
-    /** Is the metric monotonic?. */
+    /** Is the metric monotonic with suppression?. */
     private boolean           monotonic        = false;
```

Comment:
Per recommendations, retained field name for backwards compatibility in serialization.  All other changes are cool, I think.